features/networking/multus.feature

```diff
@@ -829,3 +829,40 @@ Feature: Multus-CNI related scenarios
       | /usr/sbin/ip | -d | link |
     Then the output should contain "net1"
 
+  # @author weliang@redhat.com
+  # @case_id OCP-25676
+  @admin
+  Scenario: Supported runtimeConfig/capability for MAC/IP
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    And evaluation of `project.name` is stored in the :project_name clipboard
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/Features/multus/runtimeconfig-def-ipandmac.yaml |
+      | n | <%= cb.project_name %>                                                                                                 |
+    Then the step should succeed
+
+    #Clean-up required to erase above net-attach-def after testing done
+    And admin ensures "runtimeconfig-def" network_attachment_definition is deleted from the "<%= cb.project_name %>" project after scenario
```

Comment:
just confirm. do we need this step? since the project will be deleted after the scenario. that's mean all resource will be deleted in the project

Reply:
Yep, this is not needed. If net-attach-def is created under system projects like openshift-multus, then its reqd. But anyways, this is merged now and won't impact. @weliang1 You can fix that as a part in any next PR in future